nninit.lua

```diff
@@ -270,4 +270,95 @@ nninit.sparse = function(module, tensor, sparsity)
   return module
 end
 
+--[[
+--  Aghajanyan, A. (2017)
+--  Convolution Aware Initialization
+--  arXiv preprint arXiv:1702.06295
+--]]
+nninit.convaware = function(module, tensor, options)
```

Comment:
I know it's pretty obvious, but still might be better to stick to `convolutionaware` for the name.